buildenv

```diff
@@ -24,7 +24,7 @@ zopen_check_results()
   chk="$1/$2_check.log"
 
   totalTests=198
-  testsRun=$(grep '^/bin/sh ./check-' "${chk}" | wc -l)
+  testsRun=$(grep '^/bin/bash ./check-' "${chk}" | wc -l)
```

Comment:
Should we be adding our bash as a dependency? How are you overriding the default SHELL?